packages/celotool/src/cmds/geth/start.ts

```diff
@@ -103,6 +106,21 @@ export const builder = (argv: yargs.Argv) => {
       description: 'Block Time',
       default: 1,
     })
+    .option('migrate', {
```

Comment:
What's the use case here? I mean, launching a celo testnet probably means launching more than one geth instance... but you should run migration only once...
And you should follow the steps:
 - launch all geth validator nodes
 - run migrations

So, your proposition is for us to run all geth minus the last, and then run migrations?

Reply:
> And you should follow the steps:
> 
> * launch all geth validator nodes
> * run migrations

sorry maybe i don't understand the question but this command will run a given number of geth nodes. The `migrate` switch will either migrate to the newly created network or not.

Reply:
No worries... i though this command only ran a single geth, but it runs the whole cluster. So makes sense